Casks/cursor.rb

```diff
@@ -0,0 +1,15 @@
+cask "cursor" do
+  version "0.1.3"
+  sha256 :no_check
+
+  url "https://dl.todesktop.com/230313mzl4w4u92/mac/installer/universal",
+      verified: "dl.todesktop.com/230313mzl4w4u92"
+  name "Cursor"
+  desc "Build Software. Fast. Write, edit, and chat about your code with a powerful AI"
```

Comment:
Let's remove all the marketing fluff
```suggestion
  desc "Write, edit, and chat about your code with AI"
```